packages/react-native-bpk-component-flat-list/src/theming.js

```diff
@@ -16,10 +16,12 @@
  * limitations under the License.
  */
 
+// If theming is ever expanded to support other types, this should be changed
+// to something akin to BpkButton's theming functions.
 /* @flow */
+import { makeThemePropType } from 'react-native-bpk-theming';
 
-import commonTests from './BpkSectionListItem-test.common';
+const REQUIRED_THEME_ATTRIBUTES: Array<string> = ['flatListSelectedItemColor'];
```

Comment:
nitpic: The `Array<string>` type is redundant here as flow will infer it quite easily

Reply:
It's copypasta from `BpkText` - do you think it's worth changing?

Reply:
Yeah :+1

Reply:
Cool, I'll change it everywhere then for consistency 🙌

Reply:
All done, here's the specific commit: https://github.com/Skyscanner/backpack-react-native/pull/76/commits/eb10d34f9d44348c9e333fe7488330f34aede110